terracumber_config/tf_files/Uyuni-Master-refenv-NUE.tf

```diff
@@ -0,0 +1,194 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Uyuni/job/uyuni-master-infra-reference-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='Uyuni' && run-testsuite"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/SUSE/spacewalk.git"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "master"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+// Not really used in this pipeline, as we do not send emails on success (no cucumber results)
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results Uyuni RefHead-NUE $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results RefHead-NUE: Environment setup failed"
```

Comment:
That could be something like "Results Uyuni Reference NUE: Environment setup failed"

Reply:
Fixed.